ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -265,16 +265,33 @@ pub(crate) struct Instruction {
     name: RefCell<Option<String>>,
 }
 
-impl Instruction {
+impl<'a> Instruction {
     /// Returns the operand at the specified index. Panics if the index is out of bounds.
     pub(crate) fn operand(&self, index: usize) -> &Operand {
         &self.operands[index]
     }
 
+    /// Return a slice of the remaining operands, starting from the index `from` (inclusive).
+    pub(crate) fn remaining_operands(&self, from: usize) -> &[Operand] {
+        &self.operands[from..]
+    }
+
     pub(crate) fn opcode(&self) -> Opcode {
         self.opcode
     }
 
+    /// For a call instruction, return the callee.
+    ///
+    /// Panics if the instruction isn't a call.
```

Comment:
The convention is `#! Panics` in doc strings.

Reply:
According to the rust docs, the convention is to have a "Panics" markdown level 1 heading. Is that what you meant?

Like:
```
/// Inserts an element at position `index` within the vector, shifting all
/// elements after it to the right.
///
/// # Panics
///
/// Panics if `index` is out of bounds.
```

Note sure if the `!` has significance, or is a typo.

https://rust-lang.github.io/api-guidelines/documentation.html

Reply:
Oops, I meant `# Panics`!

Reply:
Fixed in 919c1b90e93cb5e4ae6bd050a3641b3a6d6381ef